lib/PgSqlExecutor.php

```diff
@@ -214,7 +214,7 @@ public function execute(string $sql, ...$params): Promise {
      */
     public function prepare(string $sql): Promise {
         return call(function () use ($sql) {
-            yield from $this->send("pg_send_prepare", $sql, $sql);
+            yield from $this->send("pg_send_prepare", sha1($sql), $sql);
```

Comment:
Let's calculate the sha1 once and pass it to `PgSqlStatement` and store it there.